src/outfit.ts

```diff
@@ -309,6 +327,54 @@ export class Outfit {
     return this.equipSpec(thing);
   }
 
+  bjornify(target: Familiar | Familiar[]): boolean {
```

Comment:
Can you add a method comment to these? In particular mention that it doesn't add the Bjorn/throne item to the outfit, similar to modes.